go.sum

```diff
@@ -151,9 +151,8 @@ github.com/fortytw2/leaktest v1.2.0/go.mod h1:jDsjWgpAGjm2CA7WthBh/CdZYEPF31XHqu
 github.com/fortytw2/leaktest v1.3.0/go.mod h1:jDsjWgpAGjm2CA7WthBh/CdZYEPF31XHquHwclZch5g=
```

Comment:
do we need to change go.mod and go.sum file?

Reply:
Maybe, GoLand modifies go.mod and go.sum automatically...